locales/en-US/call_tool.properties

```diff
@@ -27,3 +27,8 @@ whoops_phone_number=Whoops! Make sure you enter your correct phone number.
 unknown_problems=We’re sorry, we seem to be having some trouble.
 please_retry_later=Please try again later.
 try_again=TRY AGAIN
+we_cannot_call=We can’t call that number. Please choose a country code from the list.
+hourly_limit_reached=Thanks for your calls! You’ve reached the hourly limit. Please try again in an hour.
+closed_for_business=Please note, calling is only available during business hours:
```

Comment:
Looks like this string is now: "Please visit this page during business hours to connect your call (Monday - Friday, 9:00 - 18:00 CEST)"

We're using it in two places, and I'm fine if we change "(Monday - Friday, 9:00 - 18:00 CEST)" to ": Monday - Friday, 9:00 - 18:00 CEST" when we split the strings.

I'll add this like the above to my patch, and you can then update yours, and we'll merge yours first.

Reply:
Updates to my patch here: https://github.com/mozilla/copyright/pull/141/files

Reply:
updated, looks like we’re good to go?